pkg/private/zip/build_zip.py

```diff
@@ -27,6 +27,7 @@
 # Unix dir bit and Windows dir bit. Magic from zip spec
 UNIX_DIR_BIT = 0o40000
 MSDOS_DIR_BIT = 0x10
```

Comment:
[`stat.FILE_ATTRIBUTE_DIRECTORY`](https://docs.python.org/3/library/stat.html?highlight=s_irusr#stat.FILE_ATTRIBUTE_DIRECTORY), which is equivalent?

Reply:
But that is not available in the macos python implementation. It is unfortunate that they constants vary by OS.